cdap-ui/app/cdap/components/PipelineConfigurations/PipelineConfigurations.scss

```diff
@@ -0,0 +1,459 @@
+      box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.5);
+      color: $text-color;
+      background-color: white;
+      width: $modeless-width;
+      height: $modeless-height;
+
+      .pipeline-configurations-header {
+        background-color: $modeless-header-bg-color;
+        height: 60px;
+        padding: 0;
+        border-radius: 4px 4px 0 0;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        .modeless-title {
+          color: $text-color;
+          padding-left: 15px;
+          max-width: 80%;
+          text-overflow: ellipsis;
+          white-space: nowrap;
+          overflow: hidden;
+          font-weight: 500;
+          margin: 0 auto 0 0;
+          font-size: 16px;
+          line-height: 20px;
+        }
+
+        .btn .icon-close {
+          color: $text-color;
+          width: 16px;
+          height: 16px;
+        }
+      }
+
+      .pipeline-configurations-body {
+        display: flex;
+
+        /* 60px: height of modeless box header */
+        height: calc(100% - 60px);
+
+        .configurations-side-panel {
+          border-right: 1px solid $border-color;
+          border-top: 1px solid $border-color;
+
+          .configurations-tabs {
+            .configuration-tab {
+              height: 40px;
+              display: flex;
+              align-items: center;
+              padding-left: 23px;
+              font-size: $regular-font;
+              cursor: pointer;
+              width: $side-panel-width;
+              border-bottom: 1px solid $border-color;
+
+              &:hover {
+                color: $cdap-orange;
+              }
+
+              &.active {
+                background-color: $active-bg-color;
+                color: $cdap-orange;
+                font-weight: bold;
+              }
+
+              &.disabled {
+                pointer-events: none;
+                color: lighten($text-color, 60%);
+              }
+
+              &.toggle-advanced-options {
+                padding-left: 0;
+
+                .icon-caret-right,
+                .icon-caret-down {
+                  height: 14px;
+                  width: 14px;
+                  margin: 0 1px 0 9px;
+                }
+              }
+            }
+          }
+        }
+
+        .configuration-content {
+          width: 100%;
+          margin: 16px 30px 40px 50px;
+          font-size: $regular-font;
+          position: relative;
+
+          .configuration-content-container {
+            /* 50px: height from bottom of buttons to horizontal line */
+            height: calc(100% - 50px);
+            border-bottom: 1px solid $border-color;
+
+            .configuration-step-content {
+              height: 100%;
+            }
+          }
+
+          .configuration-step-content {
+            .step-content-heading {
+              font-weight: bold;
+              margin-bottom: 16px;
+
+              .step-content-subtitle {
```

Comment:
Is this style being used anywhere?

Reply:
It's for the runtime arguments tab, which is not implemented yet in this PR